nautobot/dcim/templates/dcim/map_contact_or_team.html

```diff
@@ -0,0 +1,121 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load helpers %}
+
+{% block form %}
+    <div id="assign" class="tabcontent">
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Action</strong></div>
+            <div class="panel-body">
+                {% render_field form.action %}
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Location Data</strong></div>
+            <div class="panel-body">
+                {% render_field form.location %}
+                {% render_field form.name %}
+                {% render_field form.phone %}
+                {% render_field form.email %}
+                {% render_field form.address %}
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Contact Association</strong></div>
+            <div class="panel-body">
+                {% render_field form.contact %}
+                {% render_field form.team %}
+                {% render_field form.role %}
+                {% render_field form.status %}
+                {% render_field form.associated_object_type.as_hidden %}
+                {% render_field form.associated_object_id.as_hidden %}
```

Comment:
Still wouldn't think these fields should be needed... `associated_object_type` is always the Location ContentType, and `associated_object_id` is the same as `location`?

Reply:
Sorry I missed deleting these form fields, they are handled in the post() method now.